tests/Data/ProgrammesDb/EntityRepository/CoreEntityRepository/FindByPidsTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Tests\BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\CoreEntityRepository;
+
+use Tests\BBC\ProgrammesPagesService\AbstractDatabaseTest;
+
+/**
+ * @covers BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\CoreEntityRepository::<public>
+ */
+class FindByPidsTest extends AbstractDatabaseTest
+{
+    public function tearDown()
+    {
+        $this->getEntityManager()->getRepository('ProgrammesPagesService:CoreEntity')->clearAncestryCache();
+    }
+
+    public function testFindByPids()
+    {
+        $this->loadFixtures(['MongrelsFixture']);
+        $repo = $this->getEntityManager()->getRepository('ProgrammesPagesService:CoreEntity');
+
+        $entity = $repo->findByPids(['b00swyx1']);
```

Comment:
Probably worth adding multpiple PIDs to this array to test that, we've already got findByPid if we want a single one